pyCGM_Single/pycgmIO.py

```diff
@@ -145,6 +373,49 @@ def loadC3D(filename):
     return [data,dataunlabeled,markers]
 
 def loadCSV(filename):
+    """Open and load a CSV file of motion capture data
+
+    Keys in the returned data dictionaries are marker names, and 
+    the corresponding values are a numpy array with the associated
+    value. array([x, y, z])
+
+    Arguments
+    ---------
+    filename : str
+        File name of the CSV file to be loaded
+ 
+    Returns
+    -------
+    [motionData, unlabeledMotionData, labels] : array
+        `motionData` is a list of dict. Each dict represents one frame in 
+        the trial. `unlabeledMotionData` contains a list of dictionaries
+        of the same form as in `motionData`, but for unlabeled points. 
+        `labels` is a list of marker names.  
+
+    Examples
+    --------
+    RoboResults.csv in SampleData is used to test the output.
+
+    >>> filename = 'SampleData/Sample_2/RoboResults.csv' 
+    >>> result = loadCSV(filename)
+    >>> motionData = result[0]
+    >>> unlabeledMotionData = result[1]
+    >>> labels = result[2]
+
+    Testing for some values from data
+    >>> motionData[0]['RHNO'] #doctest: +NORMALIZE_WHITESPACE
+    array([-772.184937, -312.352295, 589.815308])
+    >>> motionData[0]['C7'] #doctest: +NORMALIZE_WHITESPACE 
+    array([-1010.098999, 3.508968, 1336.794434])
+
+    #There is no unlabeled data in frame 0
+    >>> unlabeledMotionData[0] 
```

Comment:
From our sample files, this is the only CSV file I could find to use for testing `loadCSV()`, and it has no frames with unlabeled data. I could edit the existing CSV file, or create a new one, to show what unlabeled data looks like. I added further testing in `loadC3D` to show unlabeled data in 59993_Frame_Static.c3d.

Reply:
Can you try and use the pycgm functions to: 1) load 599...c3d file 2) create some unlabeled data 3) save a csv 4) use that csv in this example.  If you do this, please make a new pull for adding the additional sample data. 

Otherwise, I think this is clearer with the explanation that there is no unlabeled data.

Reply:
I created a CSV version of Sample_Static.c3d and verified that it produces the same output using both loadC3D and loadCSV, but loadC3D had a small bug that I've created an issue for that needs fixing first before I can commit the new version of this doctest that uses Sample_Static.csv: https://github.com/cadop/pyCGM/issues/35 .